asistente_ladm_col/tests/test_toolbar.py

```diff
@@ -105,8 +105,7 @@ def test_build_boundaries_with_empty_geom(self):
         self.assertEqual(boundary_layer.selectedFeatureCount(), 1,
                          'Null features count does not match for the layer boundary layer')
 
-        boundary_layer.selectAll()
-        self.toolbar.build_boundary(db_gpkg)
+        self.toolbar.build_boundary(db_gpkg, False)
```

Comment:
This call should be simply `self.toolbar.build_boundary(db_gpkg)`.

Reply:
Adjustment made, available in commit 513f31a